content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -32,19 +32,24 @@ mitigate some future vulnerabilities too.
 {{% thirdparty-content single="true" %}}
 <!-- if adding another runtime in the future, omit the single setting -->
 
-This is a Linux only feature. In addition, support is needed in the 
+This is a Linux only feature. In addition, support is **needed** in the 
 {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}
 to use this feature with Kubernetes stateless pods:
 
-* CRI-O: v1.25 has support for user namespaces.
+* CRI-O: supports user namespaces with all Kubernetes versions >= v1.25
+* containerd: v1.7 supports user namespace with Kubernetes v1.25 and v1.26.
```

Comment:
```suggestion
* containerd: version 1.7 of CRI-O supports user namespaces for containers,
  compatible with Kubernetes {{< skew currentVersion >}}. If you are running a different
  version of Kubernetes, check the documentation for that Kubernetes release.
```

This approach works, regardless of what happens with future containerd and Kubernetes releases. I strongly recommend using it here.

Reply:
Added, changing the typo: s/of CRI-O//